src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
>Are you planning on this being a "delete all" or "delete specified nodes" feature?

Kind of answering my question at the same time, I was kind of thinking of having two: `deleteNodes(array_of_nodes)`, and `purgeNodes()`, the latter would remove all of them (probably along with the edges, as their endpoints will suddenly become invalid)